addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -984,12 +984,13 @@ options.registry.OptionsTab = options.Class.extend({
         const hues = [];
         const saturationDiffs = [];
         let oneHasNoSaturation = false;
+        const baseStyle = getComputedStyle(document.documentElement);
```

Comment:
Commit messages: don't end your title with a point + there are typos + you mentoin a commit which does not really do what you explain